services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -737,7 +743,7 @@ async def try_close_project_for_user(
     if not user_to_session_ids:
         # NOTE: depending on the garbage collector speed, it might already be removing it
```

Comment:
In case this happens, is this already handled correctly (idempotency?)

Reply:
not sure I follow. we can check that offline.